### DISCUSSION TITLE
Mini-event : Une chance surnaturelle

### DISCUSSION THREAD

**Discussion by romain22222** [Crownicles/Crownicles#1482] [Nouvelles idées]
But du mini event : faire croiser au joueur un autel, qui a la particularité d'apparaître très, très rarement (genre au moins 100 fois plus rare que les plus rares actuels)

Cet autel a la particularité de, si on décide d'y prier, peut avec une très faible chance faire apparaître un objet mythique aléatoire. Bien évidemment, sa rareté sera telle que faudra être cocu comme jamais pour avoir un item sur ce mini event, ce qui rend l'obtention d'un mythique dans ces conditions relevant du miracle

**Comment by Bombafox** [comment]
On va encore avoir des potions mdr

**Reply by BastLast** [reply]
PTDR
